docs/extend/modular-resources/configure.md

```diff
@@ -0,0 +1,149 @@
+If you are using a pre-built modular resource, make sure you install the module and determine the filename of [the module's executable](/extend/modular-resources/create/#compile-the-module-into-an-executable).
+
+Follow these steps to configure the module and modular resource:
+
+1. [Save the executable](#make-sure-viam-server-can-access-your-executable) in a location your `viam-server` instance can access.
+2. [Add a **module**](#configure-your-module) referencing this executable to the configuration of your robot.
+3. [Add a new component or service](#configure-your-modular-resource) referencing the custom resource provided by the configured **module** to the configuration of your robot.
+
+{{% alert title="Modules vs. modular resources" color="tip" %}}
+
+A configured *module* can make one or more *modular resources* available for configuration.
+
+{{% /alert %}}
+
+### Make sure `viam-server` can access your executable
+
+Ensure that your module executable is saved where the instance of `viam-server` behind your robot can read and execute it.
+
+For example, if you are running `viam-server` on an Raspberry Pi, you'll need to save the module on the Pi's filesystem.
+
+Obtain the real (absolute) path to the executable file on your computer/[board's](/components/board/) filesystem by running the following command in your terminal:
+
+``` shell
+realpath <path-to-your-module-directory>/<your-module>
+```
+
+### Configure your module
+
+To configure your new *module* on your robot, navigate to the **Config** tab of your robot's page on [the Viam app](https://app.viam.com) and click on the **Modules** subtab.
+
+The following properties are available for modules:
+
+| Name | Type | Inclusion | Description |
+| ---- | ---- | --------- | ----------- |
+`name` | string | **Required**| Name of the module you are registering. |
+`executable_path` | string | **Required**| The robot's computer's filesystem path to the module executable. |
+
+Add these properties to your module's configuration:
+
+{{< tabs >}}
+{{% tab name="Config Builder" %}}
+
+{{< imgproc src="/program/modular-resources/module-ui-config.png" alt="Creation of a new module in the Viam app config builder." resize="1000x" >}}
+
+{{% /tab %}}
+{{% tab name="JSON Template" %}}
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+  "modules": [
+    {
+      "name": "<your-module-name>",
+      "executable_path": "<path-on-your-filesystem-to/your-module-directory>/<your_executable.sh>"
+    }
+  ]
+}
+```
+
+{{% /tab %}}
+{{% /tabs %}}
+
+### Configure your modular resource
+
+Once you have configured a module as part of your robot configuration, you can add any number of the resources that module makes available to your robot by adding new components or services configured with your modular resources' new {{< glossary_tooltip term_id="api-namespace-triplet" text="subtype" >}} or [model](/extend/modular-resources/key-concepts/#models).
```

Comment:
```suggestion
Once you have configured a module as part of your robot configuration, you can add any number of the resources that module makes available to your robot by adding new components or services configured with your modular resources' new subtype or [model](/extend/modular-resources/key-concepts/#models).
```
I think the "namespace:type:subtype" tooltip isn't helpful here since we're not talking about namespacing right here.